Lib9c/Delegation/Delegatee.cs

```diff
@@ -464,5 +515,66 @@ private void TransferRemainders(LumpSumRewardsRecord record)
                 }
             }
         }
+
+        private void MigrateLumpSumRewardsRecords()
+        {
+            List<LumpSumRewardsRecord> records = new();
+            if (!(Repository.GetCurrentLumpSumRewardsRecord(this) is LumpSumRewardsRecord record))
+            {
+                return;
+            }
+
+            while (record.LastStartHeight is long lastStartHeight)
+            {
+                records.Add(record);
```

Comment:
개수가 큰 리스트일 경우 매 항목을 추가할때마다 새로운 메모리가 생성됩니다.
이를 방지하기 위해 capacity 를 지정해서 사용하면 좋을것 같습니다.
얼마만큼을 예상할 수 없다면 growSize 로 점차 늘리는 방식으로 개선해보시겠어요?
아래 수치는 예제이므로 적당히 수정해서 사용하면 됩니다.

```suggestion
            var growSize = 10;
            var capacity = 10;
            List<LumpSumRewardsRecord> records = new(capacity);
            if (!(Repository.GetCurrentLumpSumRewardsRecord(this) is LumpSumRewardsRecord record))
            {
                return;
            }

            while (record.LastStartHeight is long lastStartHeight)
            {
                if (records.Count == capacity)
                {
                    capacity += growSize;
                    records.Capacity = capacity;
                }
                records.Add(record);
```